plugin/plugin.xml

```diff
@@ -56,7 +56,7 @@
         <source-file src="src/ios/CDVRadar.m"/>
         <podspec>
             <pods use-frameworks="true">
-                <pod name="RadarSDK" spec="3.5.9"/>
+                <pod name="RadarSDK" spec="3.9.7"/>
```

Comment:
Same, let's get this to `3.9.12` -- will need:
- Exposes `user.failed_fraud` and `user.arrived_at_wrong_trip_destination` event types.
- Exposes `passed`, `inExclusionZone`, `inBufferZone`, and `distanceToBorder` on `RadarRegion`.
- Exposes `inaccurate` on `RadarFraud`.
- Improves the latency of `Radar.trackVerified()` calls.